README.md

```diff
@@ -23,8 +23,8 @@ Terracognita currently imports AWS, GCP and AzureRM cloud provider as Terraform
 Please see the following versions as follow:
 
 Providers:
- * AWS: 3.40.0
- * AzureRM: 2.58.0
+ * AWS: 4.7.0
+ * AzureRM: 2.99.0 (e1c64038e7d661d6e3a1dbfc7075f603bea9f44d)
```

Comment:
If it's an specific version then the vesrion is fine don't need the commit hash.